app/components/agenda/agenda-decisions.hbs

```diff
@@ -48,14 +48,6 @@
                         @subcase={{await agendaitem.agendaActivity.subcase}}
                         @isTableRow={{isTableRow}}
                         @agendaitem={{agendaitem}} />
-                    {{else if session.isEditor}}
```

Comment:
Ik zie niet hoe deze chenge gerelateerd is aan de handtekeningen? Anderzijds ... heel deze component komt 1 keer voor in de codebase ... Uitgecomment ...
Gezien de `print-overviews.decisions.overview`-route zelfs niet meer in `router.js` staat, denk ik dat het eens tijd is om dat in z'n geheel :arrow_right: :wastebasket: ? (Akkoord dat ik dat doe @ValenberghsSven ? )

Reply:
@MikiDi De edit hier was enkel voor een signature. Dermee dat ik die eruit had gehaald :)

Reply:
@MikiDi
- folder `pods/print-overviews/decisions/overview` kan in zijn geheel weg, die route bestaat idd niet in router.
- `components/agenda/agenda-decisions.hbs` && `agenda-decisions.js`  mag dan ook weg, worden niet meer gebruikt.